controllers/user-controller.js

```diff
@@ -112,6 +112,44 @@ const userController = {
       next(err)
     }
   },
+  getUserTweets: async (req, res, next) => {
+    try {
+      // get the tweets of a certain user (req.params.id === userId)
+      const id = req.params.id
+      const tweets = await Tweet.findAll({
+        where: { UserId: id },
+        attributes: [
+          'id', 'description', 'createdAt',
+          [
+            sequelize.literal('(SELECT COUNT(*) FROM Replies WHERE Replies.TweetId = Tweet.id)'), 'commentCount'
```

Comment:
When you make a count of replies, I think it's better to use 'replyCount' rather than 'commentCount'.